client/stories/0-general/1-userRoles.stories.mdx

```diff
@@ -0,0 +1,50 @@
+import { Meta } from "@storybook/addon-docs"
+
+<Meta title="0. ANET Stories/1. User Roles" />
+
+# Different User Roles in ANET
+
+Users of ANET can have three different roles:
+
+1. Normal, Unprivileged Users
+1. Super Users
+1. Administrators
+
+## Normal, Unprivileged Users
+
+Most of the people using ANET will be normal, unprivileged users. They can search through the
+available information, read published reports, view details about organizations, locations and such,
+and draft engagement reports to be submitted through the approval workflow.
+
+For more detailed information on users' duties, see
+[ANET User Stories](/story/1-anet-user-stories-0-introduction--page)
+
+## Super Users
+
+In addition to everything the normal, unprivileged users can do, ANET super users have a few extra
+capabilities. A super user can for example create new locations and new principals.
+
+An ANET administrator will have assigned you as super user to one or more organizations. You will be
```

Comment:
super user may not nescecary be assigned to one organisation

Reply:
Rephrased as:
```
-An ANET administrator will have assigned you as super user to one or more organizations. You will be
+When an ANET administrator has assigned you as super user to one or more organizations, you will be
 able to create new sub-organizations or positions in that organization hierarchy.
```